website/docs/users-sources/sources/social-logins/google/workspace/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Google Workspace with SAML
+sidebar_label: Workspace SAML
+tags: [integration, saml, google]
+---
+
+<span className="badge badge--primary">Support level: authentik</span>
+
+## What is Google Workspace?
+
+Google Workspace (formerly G Suite) is a collection of cloud computing, productivity and collaboration tools, software and products developed and marketed by Google.
+
+Organizations using Google Workspace allow their users to authenticate into applications using their company email addresses. This guide shows how to set up Security Assertion Markup Language (<abbr>SAML</abbr>) as the authentication method between Google Workspace and authentik.
```

Comment:
I think we might want to make this distinction a bit more easily findable, maybe a small index page for the sidebar category that says when to use either method?

Reply:
Is the distinction between using GWS as a source, versus using SAML as a middle-man between GWS and authentik? I didn't even catch the two methods when reading this paragraph. Yes, agree they need to be more clearly two separate options... instead of adding another page, I'd suggesting creating a top-level landing page where that info and other conceptual info can go.

Reply:
In general we need a sentence or two at top of every page clarifying what this page is about...

Reply:
Add this to the top-level Google page, as conceptual,please.